check_modulemd.py

```diff
@@ -24,16 +24,28 @@ def setUp(self):
         Verify required modulemd file parameter has been specified, exists,
         and can be loaded. The file name and loaded metadata are saved.
         """
+        mmd = modulemd.ModuleMetadata()
         mdfile = self.params.get('modulemd')
+        # try to use module testing farmework if possible
+        # https://pagure.io/modularity-testing-framework
+        try:
+            mtf_backend = module_framework.CommonFunctions()
```

Comment:
Yes. I've done that like that, to keep backward compatibility, how to pass params inside test.
so that actually it should be still working in actual taskotron and ready to be used by MTF.
What do you think?